### PR TITLE
Czech translations

### PR DESCRIPTION
**Pull request by YetheSamartaka**
### When merged this pull request will:

1. *Add complete Czech translations*
2. *Corrects one particular translation in danger*

### CODE REVIEW

File: addons/wp/stringtable.xml [hunk 1]
```diff
@@ -3,777 +3,916 @@
...
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_SkipCheckrounds_DisplayName">
             <English>Skip adjusting rounds</English>
+            <Czech>Přeskočit upravování střel</Czech>
             <German>Anpassen der Schüsse überspringen</German>
             <Polish>Pomiń strzały korygujące</Polish>
             <Russian>Пропустить подстройку снарядов</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_SkipCheckrounds_Tooltip">
             <English>Check this to disable initial rounds used by the fire controller to adjust rounds on target.\nSkipping this will make the barrage immediately hit on target</English>
+            <Czech>AAAAAAAA</Czech>
```
Comment (jokoho48) [inline addons/wp/stringtable.xml:90]: AAAAAAAA? i think there is the translation missing

File: addons/wp/stringtable.xml [hunk 2]
```diff
@@ -3,777 +3,916 @@
...
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_MovingCenter_Tooltip">
             <English>The hunting group will use the group leader as a center for the search pattern. Disable to have the unit use the module position instead. This creates a more defensive group</English>
+            <Czech>Nabíhající skupina použije velitele skupiny jako středobod pro pátrací vzor. Při vypnutí místo toho jednotka použije poziční modul. Toto vytvoří více obrannou skupinu</Czech>
             <German>Die jegende Gruppe benutzt den Gruppenführer als Mittelpunkt für das Suchmuster. Durch Ausschalten benutzt die Gruppe die Modulposition als Mittelpunkt. Das erzeugt eine defensivere Gruppe.</German>
             <Polish>Grupa użyje lidera jako centrum wzoru wyszukiwania. Wyłączenie opcji sprawi, że jednostka użyje zamiast tego pozycji modułu, co skutkuje tym, że grupa jest bardziej defensywna.</Polish>
             <Russian>Группа поиска будет использовать лидера группы в качестве центра схемы поиска. Отключите, чтобы устройство использовало вместо этого позицию модуля. Это создает более оборонительную группу</Russian>
-        </Key>
+        </Key>aaaaaaaaaaaaaaaaaaaaaaa
```
Comment (jokoho48) [inline addons/wp/stringtable.xml:578]: aaaaaaaaaaaaaaaaaaaaaaa?

**Review by jokoho48 — CHANGES_REQUESTED** [review]

File: addons/wp/stringtable.xml (hunk 1 above)
Comment (diwako) [reply addons/wp/stringtable.xml:90]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: addons/danger/stringtable.xml [hunk 3]
```diff
@@ -3,268 +3,314 @@
...
         <Key ID="STR_Lambs_Danger_Settings_DisableReactPlayerGroup">
             <English>Disable reaction state on player group</English>
+            <Czech>Vypne reakční stav ve skupině hráčů </Czech>
             <German>Reaktionszustand für Spielergruppe ausschalten</German>
             <Polish>Wyłącz stan reakcji dla grupy gracza</Polish>
             <Russian>Отключить состояние реакции для группы игрока</Russian>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableReactPlayerGroup_ToolTip">
             <English>Toggle Reaction phase on units in player group</English>
-            <German>Reaktionsphase für Spielergruppe ausschalten</German>
```
[the remark this answers is not in the document]
Comment (YetheSamartaka) [reply addons/danger/stringtable.xml:100]: Fixed.

File: addons/wp/stringtable.xml (hunk 1 above)
Comment (YetheSamartaka) [reply addons/wp/stringtable.xml]: Severe issue of AAAAAAAA has been fixed.

File: addons/wp/stringtable.xml (hunk 2 above)
Comment (YetheSamartaka) [reply addons/wp/stringtable.xml]: Severe issue of AAAAAAAA has been fixed.

**Comment by YetheSamartaka** [comment]
> 
> 
> Seems the recent renormalization PR made some conflicts. Once they are resolved, I think this PR is good to go for merging.

Hopefully done correctly.

**Pull request merged**